metricbeat/module/vsphere/host/host.go

```diff
@@ -240,3 +218,62 @@ func getAssetNames(ctx context.Context, pc *property.Collector, hs *mo.HostSyste
 		outputVmNames:      outputVmNames,
 	}, nil
 }
+
+func (m *HostMetricSet) getPerfMetrics(ctx context.Context, perfManager *performance.Manager, hst mo.HostSystem, metricIds []types.PerfMetricId) (map[string]interface{}, error) {
+	metricMap := make(map[string]interface{})
+	summary, err := perfManager.ProviderSummary(ctx, hst.Reference())
+	if err != nil {
+		return metricMap, fmt.Errorf("failed to get summary: %w", err)
+	}
+
+	var refreshRate = int32(m.Module().Config().Period.Seconds())
+	if summary.CurrentSupported {
+		refreshRate = summary.RefreshRate
+		if int32(m.Module().Config().Period.Seconds()) != refreshRate {
```

Comment:
Since the value is already captured in `refreshRate`, we could use that here, right?